xbmc/addons/interfaces/GUI/Window.cpp

```diff
@@ -820,6 +823,25 @@ void Interface_GUIWindow::mark_dirty_region(void* kodiBase, void* handle)
 }
 //@}
 
+/*!
+ * To get control functions
+ */
+//@{
+void* Interface_GUIWindow::getcontrol_button(void* kodiBase, void* handle, int control_id)
```

Comment:
getcontrol_button => get_control_button? (missing underscore between "get" and "control")